ReSharper.FSharp/test/data/features/quickFixes/removeNeverMatchingRule/Match expr - Multiple rules 01.fs.gold

```diff
@@ -2,6 +2,6 @@
 
 do
     match () with
+    | _ -> (){caret}
```

Comment:
Could the test have some way to tell the match clauses apart, so it's possible to reason where the caret moves? Something like:
```fsharp
match () with
| a -> ()
| b -> ()
| c -> ()
```